prow/config.yaml

```diff
@@ -66,7 +66,7 @@ presubmits:
     context: Jenkins Cross Build
     rerun_command: "@k8s-bot cross build this"
     trigger: "@k8s-bot (cross[ -])?build this"
-    run_if_changed: '(Makefile|\.sh|_(windows|linux|osx|unsupported)(_test)?\.go)$'
+    run_if_changed: '(Makefile|_(windows|linux|osx|unsupported)(_test)?\.go|/build/.+|/hack/lib/.+)$'
```

Comment:
there's no leading / in the filenames github gives us

this should probably be `'^(build/|hack/lib/)|(Makefile|_(windows|linux|osx|unsupported)(_test)?\.go)$'`